docs/api/loaders/ImageLoader.html

```diff
@@ -32,26 +32,26 @@ <h2>Example</h2>
 		loader.load(
 			// resource URL
 			'textures/skyboxsun25degtest.png',
-			// Function when resource is loaded
-			function ( image ) {
-				// do something with it
 
-				// like drawing a part of it on a canvas
+			// onLoad callback
+			function ( image ) {
+				// use the image, e.g. draw part of it on a canvas
 				var canvas = document.createElement( 'canvas' );
 				var context = canvas.getContext( '2d' );
 				context.drawImage( image, 100, 100 );
 			},
-			// Function called when download progresses
-			function ( xhr ) {
-				console.log( (xhr.loaded / xhr.total * 100) + '% loaded' );
-			},
-			// Function called when download errors
-			function ( xhr ) {
-				console.log( 'An error happened' );
+
+			// onProgress callback currently not supported
+			undefined,
+
+			// onError callback
+			function () {
+				console.error( 'An error happened.' );
 			}
 		);
 		</code>
 
+		Please note Three.js version r84 dropped support for ImageLoader progress events. If you are using a more modern version of Three.js, you may wish to use [link:https://github.com/mrdoob/three.js/issues/10439#issuecomment-275785639 one of these image loaders].
```

Comment:
I'd word this a little differently since this makes it seem optional to use an old version. Also, the 'r' in r84 stands for revision so 'version r84' is not neccessary. 

> Note: support for ImageLoader progress events was dropped in r84. See this [link:https://github.com/mrdoob/three.js/issues/10439#issuecomment-275785639 comment] for a workaround.